wrapanapi/systems/rhevm.py

```diff
@@ -451,10 +453,12 @@ def add_disk(self, storage_domain=None, size=None, interface='VIRTIO', format=No
         """
         disk_attachments_service = self.api.disk_attachments_service()
         disk_attach = types.DiskAttachment(
-            disk=types.Disk(format=types.DiskFormat(format),
+            disk=types.Disk(name=str(name),
+                            format=types.DiskFormat(format.lower()),
```

Comment:
Did API change something? why `.lower()`? Just curious.

Reply:
no, its just the method of lookup. When calling `__getattr__` on `types` classes, the all-caps attr name is used. When calling the type class to lookup the value, a lowercase string is used.  The resulting enum value is all-caps.